pkg/validator/types.go

```diff
@@ -76,4 +76,7 @@ const (
 
 	invalidResource = "Specified resource(\"%s\") value(\"%s\") for role(\"%s\") is invalid. Minimum value must be \"%s\"."
 	invalidSrcURL   = "Unable to access the specified URL(\"%s\") in file injection spec for the role (%s). error: %s."
+
+	nameLimit       = "The KD cluster name violates the name length limit. The name is a combination of cluster name + role name."
```

Comment:
Let's phrase this a little differently. Hmm. How about something like:

"When using the ClusterRole naming scheme, the total length of KD cluster name + any of its role IDs must not exceed 56 characters."

(And ideally we'd have a shared constant for that "56" amount that can be used both here as well as in validator/cluster.go, in case we need to change the limit later.)